sqlmesh/core/audit/builtin.py

```diff
@@ -68,8 +68,11 @@
 SELECT *
 FROM @this_model
 WHERE @REDUCE(
-  @criteria,
-  (l, r) -> NOT (l) OR NOT (r)
+  @EACH(
+    @criteria,
+    c -> NOT (c)
+  ),
+  (l, r) -> (l) OR (r)
```

Comment:
don't need the extra parens can just do l or r because you're already wrapping everything in NOT (x)

Reply:
I'm always pretty cautious about this. Given that parens are cheap, I tend to add them regardless especially in "code generators" like this. Removed in https://github.com/TobikoData/sqlmesh/pull/701/commits/d0986776f2d51f4112879d698deda8499376936c